File: web/app/Meal.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Meal extends Model
{
  protected $fillable=[
    'passenger_id', 'drink_id', 'food_id'
  ];

  public function passenger(){
    return $this->belongsTo('App\Passenger', 'id', 'passenger_id');
  }

  public function food(){
    return $this->hasOne('App\Food', 'id', 'food_id');
  }

  public function drink(){
    return $this->hasOne('App\Drink', 'id', 'drink_id');
  }
}

File: android/app/src/main/java/tut/flightbookingsystem/model/Meal.java
package tut.flightbookingsystem.model;

import java.io.Serializable;

public class Meal implements Serializable {
    public long id;
    public long passenger_id;
    public long drink_id;
    public long food_id;
    public Food food;
    public Drink drink;
    public String created_at;
    public String updated_at;
}
